ansible/roles/installation/nvidia-docker/tasks/install_nvidia_v2.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: 'install Docker dependencies'
```

Comment:
Start all tasks name from capital letter

Reply:
fixed in the last commit